packages/number-field/stories/number-field.stories.ts

```diff
@@ -257,16 +257,21 @@ decimals.args = {
 };
 
 export const germanDecimals = (args: StoryArgs): TemplateResult => {
+    let currentDir = 'ltr';
+    if (window.__swc_hack_knobs__) {
+        currentDir = window.__swc_hack_knobs__.defaultDirection;
+    }
```

Comment:
Can you pull this from localstorage instead of using `window.__swc_hack_knobs__`?